Makefile.am

```diff
@@ -15,8 +15,6 @@ SUBDIRS += doc/man
 endif
 .PHONY: deploy FORCE rpc-tests
 
-GZIP_ENV="-9n"
```

Comment:
This was removed upstream in https://github.com/bitcoin/bitcoin/pull/11541, which was similarly fixing automake warnings. We should just backport the first commit from that PR (as it also removes some other code we similarly don't need).